core-library/component-styling/icon-button/_icon-button.scss

```diff
@@ -89,47 +103,16 @@
         width: 16px;
         height: 16px;
       }
-    }
-
-    &.primary {
-      &.large {
-        @include primary-lg;
-      }
 
-      &.small {
-        @include primary-sm;
-      }
-
-      &.extraSmall {
-        svg {
-          width: 10px;
-        }
-      }
-
-      @include size.select(large) {
-        @include primary-lg;
-      }
-
-      @include size.select(small) {
-        @include primary-sm;
+      svg {
+        height: 16px;
```

Comment:
Can we do some exploration on if we really need to specify a height or width for the SVG? I wouldn't mind just scrapping it and having these automatically fill the span

Reply:
Chatgpt said we should set the width or height in px to resize SVG. I tried height: 100% and it didn't even show up.